students/colephalen/lesson01/assignment/inventory_management/electricAppliancesClass.py

```diff
@@ -0,0 +1,22 @@
+# Electric appliances class
+from inventoryClass import inventory
```

Comment:
One of the changes you'll want to make to get coverage to look good is to use upper case names for classes.
Ex: InventoryClass, Inventory, ElectricAppliance, etc